gradle/libs.versions.toml

```diff
@@ -149,6 +149,7 @@ commons-io = { group = "commons-io", name = "commons-io", version.ref = "commons
 caffeine = { group = "com.github.ben-manes.caffeine", name = "caffeine", version.ref = "caffeine" }
 rocksdbjni = { group = "org.rocksdb", name = "rocksdbjni", version.ref = "rocksdbjni" }
 commons-collections4 = { group = "org.apache.commons", name = "commons-collections4", version.ref = "commons-collections4" }
+iceberg-aliyun = { group = "org.apache.iceberg", name = "iceberg-aliyun", version.ref = "iceberg" }
```

Comment:
Please also update the lincese.bin to add your newly introduced jar.

Reply:
iceberg-aliyun belongs to Apache Iceberg, seems we doesn't need to add it anymore

Reply:
No, you will ship Gravitino with "iceberg-aliyun" dependency, so you need to add it to the license.bin. The aliyun dependencies you mentioned are manually added by users, so we don't have to add them to the license.bin.